README.md

```diff
@@ -187,6 +187,7 @@ The arguments are:
 * `streamBuilder` is a function that returns a subclass of the `Stream` class that supports
 the `connect` event. Typically a `net.Socket`.
 * `options` is the client connection options (see: the [connect packet](https://github.com/mcollina/mqtt-packet#connect)). Defaults:
+  * `wsOptions`: is the webSocket connection options. Default is `{}`.
```

Comment:
Can you also mentions _all_ the possible options for WebSockets? Otherwise it is not really helpful and it would create confusion (even in another section underneath).